docker/worldserver/Dockerfile

```diff
@@ -2,19 +2,30 @@ FROM ubuntu:20.04
 
 # List of timezones: http://en.wikipedia.org/wiki/List_of_tz_database_time_zones
 
-# set timezone environment variable 
+ENV CONTINUOUS_INTEGRATION=1
```

Comment:
We might want to consider changing this to something else. Maybe we should just use `DOCKER=1` instead?